symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/BdkExample.java

```diff
@@ -0,0 +1,32 @@
+package com.symphony.bdk.examples;
+
+import static com.symphony.bdk.core.config.BdkConfigLoader.loadFromSymphonyDir;
+
+import com.symphony.bdk.core.SymphonyBdk;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.InputStream;
+
+/**
+ * Base class for example creation. Config is loaded from ~/.symphony/config.yaml.
+ */
+@Slf4j
+public abstract class BdkExample {
```

Comment:
even though we are duplicating code I would keep the examples simple what was nice is that you had the entire code in one class
now to understand an example or include it in the documentation I have to understand what this class does first

Reply:
Just a proposal, removed!